src/components/LoginPage/LoginPage.js

```diff
@@ -4,18 +4,48 @@ import styled from 'styled-components';
 import { FormattedMessage } from 'react-intl';
 
 import { startLogin } from 'store/auth/actions';
-import { Button } from 'styles/elements/Button';
+import { Button, Heading } from 'styles/elements';
+import { colors, effects, fontSizes, spacing } from 'styles/base';
 
-const LoginWrapper = styled.div``;
+const LoginWrapper = styled.div`
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  justify-content: space-between;
+  &::before {
```

Comment:
spaces between pseudo elements, pseudoclasses, media improves readability, but it's up to you :)